django_app/redbox_app/redbox_core/views.py

```diff
@@ -149,14 +149,27 @@ def ingest_file(uploaded_file: UploadedFile, user: User) -> list[str]:
 @login_required
 def remove_doc_view(request, doc_id: uuid):
     file = File.objects.get(pk=doc_id)
+    errors: list[str] = []
+
     if request.method == "POST":
-        logger.info("Removing document: %s", request.POST["doc_id"])
-        file.delete()
+        api = CoreApiClient(host=settings.CORE_API_HOST, port=settings.CORE_API_PORT)
```

Comment:
this could be declared just once, maybe as a constant?

Reply:
Ultimately I'd like to inject it.